cdap-ui/app/directives/widget-container/widget-radio-group/widget-radio-group.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+angular.module(PKG.name + '.commons')
+  .directive('myRadioGroup', function() {
+    return {
+      restrict: 'E',
+      scope: {
+        model: '=ngModel',
+        config: '='
+      },
+      templateUrl: 'widget-container/widget-radio-group/widget-radio-group.html',
+      controller: function(myHelpers, $scope) {
+        let defaultValue = myHelpers.objectQuery($scope.config, 'widget-attributes', 'default') || [];
```

Comment:
why is the default value an empty array? Wouldn't radio button only have at most 1 option selected at any given time?